src/test/java/teammates/test/cases/datatransfer/AccountAttributesTest.java

```diff
@@ -133,6 +135,72 @@ public void testLegacyAccountEntityToAttributes() {
 
     }
 
+    @Test
+    public void shouldCreateDeepCopy() {
```

Comment:
Can we follow the test method naming conversion? :)
`featureUnderTest_testScenario_expectedBehavior()`